static/src/stylesheets/module/identity/_ad-prefs.scss

```diff
@@ -0,0 +1,39 @@
+.identity-ad-prefs-manager {
+
+    fieldset {
+        border: 0;
+        margin: 0;
+        padding: 0;
+        legend {
+            position: absolute;
+            left: -2000px;
```

Comment:
Are you positioning the legend offscreen to hide it? If so can you use the `u-h` class on this element instead? This'll mean screen readers will pick it up but it won't be visible.